server/v1/controllers/shows/utils/utils.js

```diff
@@ -8,6 +8,17 @@ const {
   master_time_id__byShowType,
 } = require('./utils__mongoose');
 
+function getModelForEventType(eventType) {
```

Comment:
Here's the code that returns either `db.Show` or `db.Traffic` based on the route